authenticate-with-oauth/src/main/java/com/esri/arcgismaps/sample/authenticatewithoauth/MainActivity.kt

```diff
@@ -23,16 +23,24 @@ import androidx.compose.material3.MaterialTheme
 import androidx.compose.material3.Surface
 import androidx.compose.runtime.Composable
 import androidx.lifecycle.viewmodel.compose.viewModel
+import com.arcgismaps.ArcGISEnvironment
 import com.arcgismaps.toolkit.authentication.DialogAuthenticator
+import com.arcgismaps.toolkit.authentication.signOut
 import com.esri.arcgismaps.sample.authenticatewithoauth.components.MapViewModel
 import com.esri.arcgismaps.sample.authenticatewithoauth.screens.MainScreen
 import com.esri.arcgismaps.sample.sampleslib.theme.SampleAppTheme
+import kotlinx.coroutines.runBlocking
 
 class MainActivity : ComponentActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
+        // Sign out of any portals which are already authenticated
+        runBlocking {
+            ArcGISEnvironment.authenticationManager.signOut()
+        }
+
```

Comment:
It's not recommended to use `runBlocking` in production code. The alternative here is to launch a coroutine on the Main dispatcher, then tdo the sign out followed by displaying the app:
```suggestion
        lifecycleScope.launch(Dispatchers.Main) {
            // Sign out of any portals which are already authenticated
            ArcGISEnvironment.authenticationManager.signOut()

            setContent {
                SampleAppTheme {
                    AuthenticateWithOAuthApp()
                }
            }
        }
```

Same change would need to be made to Show Portal User Info & Create And Save Map samples.